classes/controller/FrontController.php

```diff
@@ -735,6 +735,13 @@ protected function displayMaintenancePage()
     {
         if ($this->maintenance == true || !(int) Configuration::get('PS_SHOP_ENABLE')) {
             $this->maintenance = true;
+
+            $is_admin = (int) (new Cookie('psAdmin'))->id_employee;
+            $maintenance_allow_admins = (int) Configuration::get('PS_MAINTENANCE_ALLOW_ADMINS');
```

Comment:
```suggestion
            $maintenance_allow_admins = (bool) Configuration::get('PS_MAINTENANCE_ALLOW_ADMINS');
```

you used `(bool)` in some other place